blocks/identity-block/features/login/default.jsx

```diff
@@ -0,0 +1,161 @@
+import HeadlinedSubmitForm from '../../components/HeadlinedSubmitForm';
+import useIdentity from '../../components/Identity';
+
+import './styles.scss';
+
+const Login = ({ customFields, arcSite }) => {
+  let { redirectURL } = customFields;
+  const {
+    redirectToPreviousPage,
+    loggedInPageLocation,
+    forgotPasswordURL,
+    signupURL,
+  } = customFields;
+
+  const { isAdmin } = useFusionContext();
+  const { locale = 'en' } = getProperties(arcSite);
+  const phrases = getTranslatedPhrases(locale);
+
+  const { Identity, isInitialized } = useIdentity();
+
+  const [recaptchaToken, setRecaptchaToken] = useState();
+  const [recaptchaConfig, setRecaptchaConfig] = useState({
+    signinRecaptcha: false,
+    status: 'initial',
+  });
+
+  const [error, setError] = useState();
+
+  if (redirectToPreviousPage && !isServerSide()) {
+    redirectURL = document?.referrer;
+  }
+
+  useEffect(() => {
+    const getConfig = async () => {
+      await Identity.getConfig()
+        .then((response) => {
+          const {
+            signinRecaptcha,
+            recaptchaSiteKey,
+          } = response;
+
+          setRecaptchaConfig({
+            signinRecaptcha,
+            recaptchaSiteKey,
+            status: 'success',
+          });
+        })
+        .catch(() => setRecaptchaConfig({ status: 'error' }));
+    };
+
+    if (Identity) {
+      // https://redirector.arcpublishing.com/alc/docs/swagger/?url=./arc-products/arc-identity-v1.json#/Tenant_Configuration/get
+      getConfig();
+    }
+  }, [Identity]);
+
+  useEffect(() => {
+    const checkLoggedInStatus = async () => {
+      const isLoggedIn = await Identity.isLoggedIn();
+      if (isLoggedIn) {
+        window.location = loggedInPageLocation;
+      }
+    };
+    if (Identity && !isAdmin) {
+      checkLoggedInStatus();
+    }
+  }, [Identity, loggedInPageLocation, isAdmin]);
+
+  if (!isInitialized) {
+    return null;
+  }
+
+  return (
+    <>
+      <HeadlinedSubmitForm
+        headline={phrases.t('identity-block.log-in')}
+        buttonLabel={phrases.t('identity-block.log-in')}
+        onSubmit={({ email, password }) => Identity
+          .login(email, password, { rememberMe: true, recaptchaToken })
+          .then(() => { window.location = redirectURL; })
+          .catch(() => setError(phrases.t('identity-block.login-form-error')))}
+        formErrorText={error}
+      >
+        <FormInputField
+          label={phrases.t('identity-block.email')}
+          name="email"
+          required
+          showDefaultError={false}
+          type={FIELD_TYPES.EMAIL}
+          validationErrorMessage={phrases.t('identity-block.email-requirements')}
+        />
+        <FormInputField
+          label={phrases.t('identity-block.password')}
+          name="password"
+          required
+          showDefaultError={false}
+          type={FIELD_TYPES.PASSWORD}
+          validationErrorMessage={phrases.t('identity-block.password-requirements')}
```

Comment:
not sure what the standard on password requirements for login. should we even validate this? 

doesn't look like others do 
![Screen Shot 2021-09-13 at 15 16 40](https://user-images.githubusercontent.com/5950956/133150406-6b73549e-8017-415b-afbe-56f2793b164e.png)

![Screen Shot 2021-09-13 at 15 17 18](https://user-images.githubusercontent.com/5950956/133150522-20347b2e-e310-48f2-9ed5-b264540d8d13.png)

Reply:
We should _not_ validate.  If we change the rule later, that will lock people out of logging in.

Reply:
Thinking this should also apply to the login field too... checking.

Reply:
AC modified